ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueConsumption.scala

```diff
@@ -123,23 +145,14 @@ object KeyValueConsumption {
   def logEntryToAsyncResponse(
       entryId: DamlLogEntryId,
       entry: DamlLogEntry,
-      participantId: String): Option[AsyncResponse] = {
+      participantId: ParticipantId): Option[AsyncResponse] = {
 
     entry.getPayloadCase match {
```

Comment:
To improve readability, this may now be rewritten with `PartialFunction.condOpt` if we don't explicitly want to have a `case` for each `PayloadCase` (as most of the responses will be `None`):
```scala
PartialFunction.condOpt(entry.getPayloadCase) {
  case DamlLogEntry.PayloadCase.PARTY_ALLOCATION_ENTRY if participantId == entry.getPartyAllocationEntry.getParticipantId =>
    PartyAllocationResponse(
              entry.getPartyAllocationEntry.getSubmissionId,
              ...
    )
  case DamlLogEntry.PayloadCase.PARTY_ALLOCATION_REJECTION_ENTRY if participantId == entry.getPartyAllocationEntry.getParticipantId =>
    partyRejectionEntryToAsyncResponse(entry.getPartyAllocationRejectionEntry)
  ...
}
```

Reply:
Thanks - will try this in future.  In this case this function will soon go away once the party allocation entries are moved to the new style so will leave it for the moment and remove once the party allocation entries are gone

Reply:
this method is now deleted